CHANGELOG.md

```diff
@@ -254,7 +267,6 @@ This is the first release since the module was migrated to the Vox Pupuli `puppe
 **Merged pull requests:**
 
 - For folks that do not use redis to cache to disk [\#144](https://github.com/voxpupuli/puppet-redis/pull/144) ([petems](https://github.com/petems))
-- Revert "Changes templates to use scope" [\#143](https://github.com/voxpupuli/puppet-redis/pull/143) ([petems](https://github.com/petems))
```

Comment:
This change was definitely part of 1.2.4 and not 3.0.0.  What is causing this??? AGHHH!!!

Reply:
https://github.com/github-changelog-generator/github-changelog-generator/pull/729 has fixed this issue.  I've been able to repeatably generate the 'correct' changelog more than 10 times.  Using gcg master branch, the changelog generated for this project is different on most runs.